src/base/utils/fs.cpp

```diff
@@ -355,6 +355,13 @@ bool Utils::Fs::sameFileNames(const QString &first, const QString &second)
 #endif
 }
 
+bool Utils::Fs::isFileInFolder(const QString &file_path)
```

Comment:
>My fix is just in a few lines

In this case, please spare the world from some more unnecessary lines of code. This function should not be here. Besides, it can be done easier: `m_torrentInfo.filePath(0).contains('/')` in addnewtorrentdialog.cpp